Create video.json.md (#907)
pyvideo/data: .schemas/video.json.md

>  # Schema for Video Record.

Reviewer: ```suggestion
# Schema for Video Record.
```

> [PyVideo.org](https://pyvideo.org) runs on github and every video is described by some json code based on the [official schema](https://github.com/pyvideo/data/blob/master/.schemas/video.json).

Reviewer: ```suggestion
The data behind [PyVideo.org](https://pyvideo.org) is hosted on Github and every video is described by a JSON file which adheres to the [official schema](https://github.com/pyvideo/data/blob/master/.schemas/video.json).
```

  "recorded": "2019-09-22",
  "speakers": [
    "Takanori Suzuki"
  ],
  "thumbnail_url": "https://i.ytimg.com/vi/XGHR4D8_fjQ/hqdefault.jpg",
  "title": "Automate the Boring Stuff with Slackbot",
  "videos": [
    {
      "type": "youtube",
      "url": "https://www.youtube.com/watch?v=XGHR4D8_fjQ"
    }
  ]
}
```

> ## copyright_text

Reviewer: ```suggestion
## `copyright_text` (optional)
```
Let's make all the headers look like this.

> This parameter is optional. It should have the text of the license under which the media is being shared or a link to a license if it exists. The copyright_text may also be "null"

Reviewer: And then we don't need to put whether it's optional or not in the description.

> #### examples

Reviewer: ```suggestion
#### Examples
```
Here and elsewhere

```json
{
    "copyright_text" : null
}
{

>     "copyright_text" : "THIS VIDEO AND THE SOFTWARE IT DESCRIBES IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE."

Reviewer: ```suggestion
    "copyright_text" : "THIS VIDEO AND THE SOFTWARE IT DESCRIBES IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE."
```

}
{
    "copyright_text" : "https://opensource.org/licenses/AGPL-3.0"
}

> ```

Reviewer: Let's break each of these into their own code block:

```json
{
    "copyright_text" : null
}
```
```json
{
    "copyright_text" : "THIS VIDEO AND THE SOFTWARE IT DESCRIBES IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE."
}
```
```json
{
    "copyright_text" : "https://opensource.org/licenses/AGPL-3.0"
}
```

### examples
```json
{
    "language" : "eng"
}
{
    "language" : "fra"
}
{
    "language" : "deu"
}
```
## quality_notes
This is an optional parameter. You can use this section to provide some notes on the quality of the video.
### examples
```json
{
    "quality_notes" : null
}
{
    "quality_notes" : "Great audio and sound"
}
{
    "quality_notes" : "Sound is poor for the first 13 minutes but is better after that"
}
```
## recorded
This is a required parameter. Provide the date the video was recorded in ISO-8601 format i.e. `(YYYY-MM-DD[Thh:mm:ss[+hh:mm]])`
### examples
```json
{
    "recorded" : null
}
{
    "recorded" : "2008-09-06"
}
    "recorded" : "2019-08-10T13:33:30+05:30"
}
```
## related_urls
This is an optional parameter. This is a list of URLs to resources that are related to the video.
### examples
```json
{
    "related_urls": [
                      "https://www.tensorflow.org/tutorials",
                      "https://bit.ly/tf-ws1",
                      "https://bit.ly/tf-ws3",
                      "https://bit.ly/tf-ws4",
                      "https://bit.ly/tf-ws4a"
                      ]

> }                     

Reviewer: I think the autoformatter would turn this into:

```json
{
    "related_urls": [
        "https://www.tensorflow.org/tutorials",
        "https://bit.ly/tf-ws1",
        "https://bit.ly/tf-ws3",
        "https://bit.ly/tf-ws4",
        "https://bit.ly/tf-ws4a"
    ]
} 
```

{
      "related_urls": [
    {
      "label": "Conference schedule",
      "url": "https://us.pycon.org/2018/schedule/talks/"
    },
    {
      "label": "Conference slides (Github)",
      "url": "https://github.com/PyCon/2018-slides"
    },
    {
      "label": "Conference slides (SpeakerDeck)",
      "url": "https://speakerdeck.com/pycon2018"
    }
  ],
}
```
## slug
This is an optional parameter. Provide a web safe [slug](https://en.wikipedia.org/wiki/Clean_URL#Slug) of the video.
### example
```json
{
    "slug" : "what-happened-at-aaronswhack"
}
```
## speakers
This is a required parameter. This is an array of the speakers in the video arranged in alphabetical order based on the last name of the speakers.
### examples
```json
{
    "speakers": ["Eugenio Llanos"]
}
{
    "speakers": [ "James Bednar", 
                  "Philipp Rudiger", 
                  "Julia Signell",
                  "Jean-Luc Stevens"

>                 ],

Reviewer: Same comment here about indentation.
